### DISCUSSION TITLE
How to make category tree show only parent?

### DISCUSSION THREAD

**Discussion by hellodit** [bagisto/bagisto#9683] [Q&A] [answered]
Hi guys, I want to make my category tree by default show only the root parent level and close all the child levels

The default like this 
![image](https://github.com/bagisto/bagisto/assets/32544683/cac8ca41-2d38-493f-b9fb-589c0e1228da)

I want to make it like this, the tree is closed 
![image](https://github.com/bagisto/bagisto/assets/32544683/911f3ffa-8a6f-4cdd-9e88-a2b2edb2ab38)
 

What should I modify?

**Comment by devansh-webkul** [comment]
Hello,

Currently there is no way, but we can provide the feature for collapse all and expand all. Currently you can overide the behavior.

**Comment by devansh-webkul** [comment]
I have opened an issue and thanks for reporting this.

**Comment by devansh-webkul** [comment, marked as answer]
For a quick fix you can change here,
https://github.com/bagisto/bagisto/blob/master/packages/Webkul/Admin/src/Resources/views/components/tree/view.blade.php#L203

Just remove the 'active' class for here. Rest it will handle automatically.

**Reply by hellodit** [reply]
Thanks 👍
